independent-projects/bootstrap/maven-plugin/pom.xml

```diff
@@ -15,7 +15,7 @@
             <plugin>
                 <groupId>org.codehaus.plexus</groupId>
                 <artifactId>plexus-component-metadata</artifactId>
-                <version>2.1.0</version>
+                <version>2.1.1</version>
```

Comment:
2.1.1 is fluke due due fluke qdox version. Go back to 2.1.0

As there is no newer version, while fix is committed https://github.com/codehaus-plexus/plexus-containers/commit/88f833bcc8ce843441fca1de669989c5ade366b1

Reply:
Thanks!

Reply:
Actually, any reason why plexus annos are used here? This project is already to be archived https://github.com/codehaus-plexus/plexus-containers/issues/59 w/ plexus-shim as drop in replacement (only to maintain backward compat, NOT to create NEW code with it).

Reply:
Will prep a PR that gets rid of ancient plexus annos in favor of JSR330 as it is unrelated to this PR

Reply:
Thanks @cstamas!